### DISCUSSION TITLE
Question about Remote URL whitetlist

### DISCUSSION THREAD

**Discussion by MannixMD** [mach64/trakt-scrobbler#143] [Q&A] [answered]
Can someone shed some light on my oblivious soul and tell me what it is and how to use it. Tell me if I'm wrong but with this I can add a hosting site like mega to the whitelist and the scrobbler should pick it up when i stream something in the monitored player ?

**Comment by iamkroot** [comment, marked as answer]
Two things:
1. Our app only cares about the media info (like title, season, etc.). Your media player is the one responsible for actually playing the media. Players like MPV and VLC support playing a video from any remote URL, where the data is directly accessible. But Mega (and many other file hosting sites) has its own downloader, which means the player can't play the file directly.
2. The scrobbler can only extract media information from the file path/name (except for Plex, which has its own library of metadata). So when using remote playback, the URL still needs to have the full file name, just like it would have on your local drive. Since mega only has some random ID in the URL, this means our app won't be able to figure out the media info, even if you could play the file on VLC/MPV.

**Comment by MannixMD** [comment]
I see thank you for swift response and clarification :)
